api/v1alpha1/groupversion_info.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The KCP Authors.
```

Comment:
Just need at this author header back to all of the files and and this PR is good to go.